src/ontology/reports/sync-synonym/doid.synonyms.confirmed.robot.tsv

```diff
@@ -2316,7 +2283,6 @@ MONDO:0002508	gingivitis	oio:hasExactSynonym		gingivitis			DOID:3087	gingivitis
 MONDO:0002512	papillary adenocarcinoma	oio:hasExactSynonym		papillary adenocarcinoma			DOID:3112	papillary adenocarcinoma					papillary adenocarcinoma	DOID:3112										
 MONDO:0002513	kidney benign neoplasm	oio:hasExactSynonym		kidney benign neoplasm			DOID:3116	kidney benign neoplasm					kidney benign neoplasm	DOID:3116										
 MONDO:0002513	kidney benign neoplasm	oio:hasExactSynonym		renal and ureteral tumor			DOID:3116	kidney benign neoplasm					renal and ureteral tumor	DOID:3116										
-MONDO:0002514	hepatobiliary neoplasm	oio:hasExactSynonym		hepatobiliary tumors			DOID:3117	hepatobiliary benign neoplasm					hepatobiliary tumors	DOID:3117										
```

Comment:
## Case 1 of 2 removed
As shown in the QC error in #687, this pair (`MONDO:0002514`, `DOID:3117`) had no exact mapping. Now it is removed.